### ISSUE TITLE
doctest-in-workspace flag is no longer required

### ISSUE THREAD

**Issue by danieleades** [taiki-e/cargo-llvm-cov#288]
when running the following command-

```sh
cargo llvm-cov --doctests 
```

i see this warning-

```
flag `-Z doctest-in-workspace` has been stabilized in the 1.72 release, and is no longer necessary
  The doctest-in-workspace feature is now always enabled.
```

if i take that at face value, does that mean the flag can be removed?
is the 'doctests' making use of any other unstable features or can that now be stabilised too?

**Comment by taiki-e** [comment]
Thanks. Addressed in v0.5.21.

> is the 'doctests' making use of any other unstable features or can that now be stabilised too?

instrument-coverage for doctests requires unstable `--persist-doctests` too.
https://doc.rust-lang.org/nightly/rustc/instrument-coverage.html#including-doc-tests

**Comment by pksunkara** [comment]
This new release 0.5.21 is not working with stable rust anymore.

```
$ cargo llvm-cov --all --all-features --lcov --output-path lcov.info
error: the `-Z` flag is only accepted on the nightly channel of Cargo, but this is the `stable` channel
See https://doc.rust-lang.org/book/appendix-07-nightly-rust.html for more information about Rust release channels.
error: process didn't exit successfully: `/home/pksunkara/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/bin/cargo test -Z doctest-in-workspace --tests --manifest-path /home/pksunkara/Coding/project/Cargo.toml --target-dir /home/pksunkara/Coding/project/target/llvm-cov-target --all --all-features` (exit status: 101)
```

0.5.20 works though.

**Comment by taiki-e** [comment]
Oh, sorry for the breakage! Fixed in 0.5.22 and yanked 0.5.21.
